lib/freight/apt.sh

```diff
@@ -79,7 +79,7 @@ apt_filesize() {
 # including all packages read from `stdin`.
 apt_cache() {
 	REL_DATE="$(LC_ALL=en_US date '+%a, %d %b %Y %H:%M:%S %Z')"
-	VALID_DATE="Fri, 31 Nov 2038 00:00:00 JST"
+	VALID_DATE="Fri, 30 Nov 2038 00:00:00 JST"
```

Comment:
In that turn, `Fri` could be changed to `Tue`, to appease stricter checks (I have no idea if there's any software which does this...)